string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,22 @@
+/* exported isVowel */
+
+/* - First check if the character is uppercased or lowercased
+   - Then check to see if the letter is a vowel
+        - if it is a vowel, give back true
+        - if it is not a vowel, give back false
+*/
+
+function isVowel(char) {
+  if (char >= 'A' && char <= 'Z') {
+    if (char === 'A' || char === 'E' || char === 'I' || char === 'O' ||
+    char === 'U') {
+      return true;
+    }
+  } else if (char >= 'a' && char <= 'z') {
+    if (char === 'a' || char === 'e' || char === 'i' || char === 'o' ||
+      char === 'u') {
+      return true;
+    }
+  }
```

Comment:
If you just make the character uppercase or lowercase before the conditional, you can just check one set of characters instead of having to check both uppercase and lowercase